src/seer/trend_detection/trend_detector.py

```diff
@@ -18,23 +18,29 @@
 from seer.trend_detection.detectors.cusum_detection import CUSUMDetector
 
 
-def find_trends(txns_data, sort_function, zerofilled, allow_midpoint, trend_perc=0.1, pval=0.01):
+def find_trends(
+    txns_data,
+    sort_function,
+    allow_midpoint,
+    min_pct_change,
+    min_ms_change,
```

Comment:
And update this argument name too

```suggestion
    min_change,
```